WordPress/Classes/ViewRelated/Media/MediaLibraryPickerDataSource.m

```diff
@@ -532,6 +532,53 @@ - (WPMediaRequestID)imageWithSize:(CGSize)size completionHandler:(WPMediaImageBl
     return [self.mediaID intValue];
 }
 
+- (WPMediaRequestID)videoAssetWithCompletionHandler:(WPMediaAssetBlock)completionHandler
+{
```

Comment:
1) I think this method would benefit from some comments.
2) Since the ifs seem to be exclusive, can they be re-organized in a way that makes that clearer?